en_US/multi-tenancy/namespace.md

```diff
@@ -0,0 +1,256 @@
+
+```hcl
+mqtt.client_attrs_init = [{expression = username, set_as_attr = tns}]
+```
+
+### Methods for Creating a Namespace
+
+There are two methods for creating namespaces: explicit creation and automatic creation.
+
+1. **Explicit Namespace Creation**
+   You can manually create a namespace through the Dashboard or REST API. Explicitly created namespaces can be directly managed, edited, and deleted.
+
+   **Use Case**: This method is ideal when you need full control over which namespaces exist and wish to manage them individually.
+
+2. **Automatic Namespace Creation via EMQX Extracting `client_attrs.tns`**
+   When clients connect, EMQX can automatically create namespaces based on the client’s `client_attrs.tns` attribute. This method is usually used in scenarios where manual namespace creation is not necessary and is suitable for large-scale automated deployments.
+
+   **Use Case**: This approach works best for environments where clients connect dynamically, and each tenant or client requires an automatically generated, independent namespace.
+
+   ::: tip Note
+
+   Automatically created namespaces cannot be edited in the Dashboard. They are generated either through configuration files or automatically based on client metadata.
+
+   :::
+
+With these two methods, you can choose the most appropriate way to create namespaces based on your needs. Explicit creation is ideal for environments requiring strict control, while automatic creation is suitable for dynamic environments with reduced manual intervention.
+
+## Quick Start: Configure Rate Limiter per Namespace
+
+You can configure per-namespace rate limiters to control traffic and message flow for specific client groups. These limiters work in conjunction with EMQX's existing rate limiters for zones and listeners, depending on the type of limiter configured.
+
+### Limiter Types
+
+In a managed namespace, there are two types of rate limiters:
+
+**Tenant rate limiters**: Assign tokens that are **shared** across all clients within a namespace (NS). When this type of limiter is configured, it composes with any existing zone-level rate limiters, meaning both the zone and the namespace tenant rate limiters apply to clients simultaneously.
+
+**Client rate limiters**: Assign tokens that are **exclusive** to each client within the NS. When this type of limiter is configured, it replaces any existing listener-level rate limiters, meaning the listener rate limiters are ignored while the namespace client limiter takes effect.
+
+Both limiter types can define limits for:
+
+- **Message rate limits**: the maximum number of messages a client or tenant can publish over a given time period
+- **Byte throughput limits**: the maximum allowed size for message payloads over time
+- **Subscription rate limits**: the maximum number of topics a client can subscribe to within a specified time window
```

Comment:
The original form was correct: messages rate limit limits subscriptions too.

```suggestion
```

Reply:
Sorry, either that changed recently, or I confused the situations when subscriptions could be limited. 

`SUBSCRIBE` packets are actually _not_ rate limited.  :see_no_evil: :thinking: